### ISSUE TITLE
Best Buy 3090 FE drop on Oct 9 2pm CST not detected

### ISSUE THREAD

**Issue by AndyinMission** [jef/nvidia-snatcher#488]
### Description

Best buy had a drop of 3090 FE cards around 2pm on Oct 9th that lasted ~15 min.   NVIDIA snatcher did not detect this event.

### Possible solution

URL or page elements changed?

**Comment by Chais82** [comment]
I wanted to say thank you to the developer. I managed to grab both a 3080 and a 3090 from best buy.

**Comment by Chais82** [comment]
> I wanted to say thank you to the developer. I managed to grab both a 3080 and a 3090 from best buy.

However, it only detected the 3080 like this post is stating.

**Comment by michaelv2** [comment]
> I wanted to say thank you to the developer. I managed to grab both a 3080 and a 3090 from best buy.

I had the good fortune of actually being in front of my system when the alert came up at 2:00PM ET, but when I went to their website the 3080 FE cards were still all listed as "Sold Out" (I tried refreshing several times from 2:00 - 2:10 or so), and because I kept getting alerts I thought it was a bug! Might I ask if there was some "trick" you used, or perhaps the stock / availability was somehow regional? Several people on Reddit mentioned being able to add one to their cart even 15 minutes after the drop. Thanks in advance!
